code/game/machinery/kitchen/cooking_machines/_appliance.dm

```diff
@@ -112,7 +112,7 @@
 /obj/machinery/appliance/verb/toggle_power()
 	set name = "Toggle Power"
 	set category = null
```

Comment:
Why not make robots use the AI verb and make this `src in view(1)`? That'll reduce verb clutter further.
Also, probably should remove `set category = null` here.